lib/main_common.pm

```diff
@@ -1432,6 +1433,11 @@ sub load_extra_tests_desktop {
             # only scheduled on gnome and was developed only for gnome but no
             # special reason should prevent it to be scheduled in another DE.
             loadtest 'x11/steam' if check_var('ARCH', 'x86_64');
+            # on SLE thunderbird is tested in qam-regression-message, but we
+            # don't have such test suite on openSUSE.
```

Comment:
Actually I recommend to simply add that test suite. Better to not extend the existing "extra_tests" but define new ones as the scenarios are already hard to handle during test review.

Reply:
+1